requirements.txt

```
@@ -24,6 +24,7 @@ Pillow>=6.0.0,<8.4.0
 pyahocorasick>=1.4,<1.5
 pycairo>=1.15.3
 pygobject>=3.27.0
+pyicumessageformat>=1.0.0
```

```suggestion
pyicumessageformat>=1.0.0,<1.1.0
```

Let's make the dependency strict to avoid issues on API change in future.